src/Microsoft.DotNet.ProjectJsonMigration/ProjectMigrator.cs

```diff
@@ -99,6 +97,11 @@ private void VerifyProject(IEnumerable<ProjectContext> projectContexts, string p
             }
         }
 
+        private string FormatDiagnosticMessage(DiagnosticMessage d)
```

Comment:
@livarcocc There's more work here with #4164 , I'll leave the refactoring til then